### ISSUE TITLE
Negative Values Y Axis

### ISSUE THREAD

**Issue by aphoustons** [mindspank/WaterfallChart#1]
Hi, thanks for building the Waterfall chart for Qlik Sense, is there any chance you could modify the Y Axis to show negative values. At present the chart doesn't scale for values below 0 so we can't use it in our business models. Is this possible to update? 

Many thanks,
Alex

**Comment by mindspank** [comment]
Yes there is some work to make it scale below 0 but I will have a look at it as soon as possible.

**Comment by aphoustons** [comment]
Great thank you!

**Comment by jpvieira88** [comment]
Hi, i am having the same issue. Is there any solution / alternative?

thanks

**Comment by keithchap** [comment]
This issue is preventing us from using this as a solution also. Is there any update on a solution?
Thank you-

**Comment by mindspank** [comment]
@keithchap I haven't had time to work on it but pull requests are highly encouraged so if you do work on it  please contribute back
